typings/codeceptjs.d.ts

```diff
@@ -1,33 +1,36 @@
 
-type ICodeceptCallback = (i: CodeceptJS.I, config:any, loginPage:any, caseListPage:any, createCasePage:any, addEventSummaryPage:any, caseViewPage:any, selectHearingPage:any, enterGroundsPage:any, enterFactorsAffectingParentingPage:any, enterInternationalElementsPage:any, enterRiskAndHarmToChildPage:any, uploadDocumentsPage:any, enterApplicantPage:any, enterChildrenPage:any, enterOtherProceedingsPage:any, attendingHearingPage:any, enterAllocationProposalPage:any, enterRespondentsPage:any, enterOthersPage:any, ordersNeededPage:any, enterFamilyManPage:any, changeCaseNamePage:any, deleteApplicationPage:any) => void;
+type ICodeceptCallback = (i: CodeceptJS.I, config:CodeceptJS.config, loginPage:CodeceptJS.loginPage, caseListPage:CodeceptJS.caseListPage, createCasePage:CodeceptJS.createCasePage, eventSummaryPage:CodeceptJS.eventSummaryPage, caseViewPage:CodeceptJS.caseViewPage, selectHearingPage:CodeceptJS.selectHearingPage, enterGroundsPage:CodeceptJS.enterGroundsPage, enterFactorsAffectingParentingPage:CodeceptJS.enterFactorsAffectingParentingPage, enterInternationalElementsPage:CodeceptJS.enterInternationalElementsPage, enterRiskAndHarmToChildPage:CodeceptJS.enterRiskAndHarmToChildPage, uploadDocumentsPage:CodeceptJS.uploadDocumentsPage, enterApplicantPage:CodeceptJS.enterApplicantPage, enterChildrenPage:CodeceptJS.enterChildrenPage, enterOtherProceedingsPage:CodeceptJS.enterOtherProceedingsPage, attendingHearingPage:CodeceptJS.attendingHearingPage, enterAllocationProposalPage:CodeceptJS.enterAllocationProposalPage, enterRespondentsPage:CodeceptJS.enterRespondentsPage, enterOthersPage:CodeceptJS.enterOthersPage, ordersNeededPage:CodeceptJS.ordersNeededPage, enterFamilyManPage:CodeceptJS.enterFamilyManPage, changeCaseNamePage:CodeceptJS.changeCaseNamePage, submitApplicationPage:CodeceptJS.submitApplicationPage, sendToGatekeeperPage:CodeceptJS.sendToGatekeeperPage, standardDirectionsPage:CodeceptJS.standardDirectionsPage, deleteApplicationPage:CodeceptJS.deleteApplicationPage, ...args: any) => void;
```

Comment:
lots of changes to this file. Intentional? If so I would revert and add as a separate PR

Reply:
I will try to remove the typings and see if it works as separate commit, the problem is I have added new function to steps and didn't want to have compilation error.